server/routes/cart/cart.js

```diff
@@ -38,4 +36,25 @@ router.get(
   }
 );
 
+// ******************delete address******************//
```

Comment:
why does the comment say "delete address" ?

Reply:
It's by mistake

Reply:
I changed it